### ISSUE TITLE
Client.get_policy() returns UnimplementedError and Client.create_policy() does not create policy version

### ISSUE THREAD

**Issue by surfaslt** [spulec/moto#1876]
Client.get_policy() and Client.get_policy_version() returns UnimplementedError even though it is implemented on moto backend.

A workaround for that is to call the methods from directly by importing and using moto backend.

Please fix this so that people do not need to mock out these functions with "adaptor" functions that would call the same method just directly from backend.

**Comment by spulec** [comment]
Can you give some example code that doesn't work?

We have a [test here](https://github.com/spulec/moto/blob/c035348ae440f35134af05695153f5f0b6cc171d/tests/test_iam/test_iam.py#L313-L316) for those methods.

**Comment by surfaslt** [comment]
For get_policy:

This works:
`client = iam_backends['global']
        response = client.get_policy(attachedpolicy['PolicyName'])`
However, this returns unimplemented error:
`
client = boto3.client('iam')
        response = client.get_policy(attachedpolicy['PolicyName'])`

As for get_policy_version it seems like my problem was that after doing:
`
boto3.client('iam').create_policy(<...>)
`
I also needed
`
boto3.client('iam').create_policy_version(<...>)
`
as otherwise it was returning no such entity found (or equivalent)

P.s. I am using Moto v1.3.6 and I know that according to https://github.com/spulec/moto/pull/1759 I should not have needed to manually create the policy version, but for some reason the list of versions was `[]` if I didn't.

**Comment by spulec** [comment]
I'm still having trouble figuring out what is going on.

[This test](https://github.com/spulec/moto/blob/c035348ae440f35134af05695153f5f0b6cc171d/tests/test_iam/test_iam.py#L289-L297) uses the code in the way you describe as not working. Can you share some more code so I can try to figure out what is different in your scenario from that test?

**Comment by surfaslt** [comment]
I have done a small test with just these two lines on their own and indeed for the second line I got `NotImplementedError: The get_policy action has not been implemented`. I have tried changing requirements to `boto3==1.7.84` and `botocore==1.10.84` but got same results. 
```
client = boto3.client('iam')
response = client.get_policy(PolicyArn='arn:aws:iam::123456789012:some-project')
```

**Comment by spulec** [comment]
Okay, sounds like this is fixed with `1.3.7`. Feel free to reopen if not.
